src/test/java/com/gooddata/warehouse/WarehouseServiceTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ */
+package com.gooddata.warehouse;
+
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.client.ClientHttpResponse;
+import org.springframework.web.client.RequestCallback;
+import org.springframework.web.client.ResponseExtractor;
+import org.springframework.web.client.RestTemplate;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class WarehouseServiceTest {
+    private WarehouseService service;
+
+    private WarehouseS3Credentials s3Credentials = new WarehouseS3Credentials("region", "accessKey", "secretKey");
+
+    @Mock
+    private Warehouse warehouse;
+    @Mock
+    private RestTemplate restTemplate;
+
+
```

Comment:
Extra line

Reply:
removed, added another unit test instead of it